src/includes/class-wordlift-post-to-jsonld-converter.php

```diff
@@ -167,8 +167,32 @@ public function convert( $post_id, &$references = array(), &$references_infos =
 		// Set the publisher.
 		$this->set_publisher( $jsonld );
 
-		// Finally set the author.
-		$jsonld['author'] = $this->get_author( $post->post_author, $references );
+		/**
+		 * Call the `wl_post_jsonld_author` filter.
+		 * 
+		 * This filter checks if there are co-authors or a single author and
+		 * returns a JSON-LD fragment for the author(s).
+		 * 
+		 * @param array $value {
+		 * 
+		 * @type array $jsonld The JSON-LD structure.
+		 * @type int[] $references An array of post IDs.
+		 * }
+		 * 
+		 * @param int $post_id The {@link WP_Post} `id`.
+		 * 
+		 * @since 3.51.4
+		 * 
+		 * @see https://www.geeklab.info/2010/04/wordpress-pass-variables-by-reference-with-apply_filter/
+		 */
+		$jsonld['author'] = apply_filters(
+			'wl_jsonld_author',
+			array(
+				'author' 	 => $this->get_author( $post->post_author, $references ),
+				'references' => $references
```

Comment:
I don't think `$references` would work as an output in this case.

Check src/includes/class-wordlift-entity-post-to-jsonld-converter.php:L123 to see how we do it in a similar case:

```
		/**
		 * Call the `wl_post_jsonld_array` filter. This filter allows 3rd parties to also modify the references.
		 *
		 * @param array $value {
		 *
		 * @type array $jsonld The JSON-LD structure.
		 * @type int[] $references An array of post IDs.
		 * }
		 * @since 3.25.0
		 *
		 * @see https://www.geeklab.info/2010/04/wordpress-pass-variables-by-reference-with-apply_filter/
		 *
		 * @api
		 */
		$ret_val    = apply_filters(
			'wl_entity_jsonld_array',
			array(
				'jsonld'     => $jsonld,
				'references' => $references,
			),
			$post_id
		);
		$jsonld     = $ret_val['jsonld'];
		$references = $ret_val['references'];
```